lib/utils/file.js

```diff
@@ -0,0 +1,17 @@
+// Based on https://github.com/kevva/executable
+// Since this module doesn't expose the function to check stat mode only,
+// his logic is pasted here.
+// Opened an issue and a pull request about it,
+// to maybe switch to module in the future:
+// Issue: https://github.com/kevva/executable/issues/9
+// PR: https://github.com/kevva/executable/pull/10
+
+export function isExecutable (fileStat) {
+  if (process.platform === 'win32') return true;
```

Comment:
Now opening a txt on windows would simply return 'Bad command...', but you can open an executable and it would start automatically. Otherwise, if returning false, both opening a txt or an executable would insert the path without adding the "\n".
I don't know, maybe you're right and return always false is a better approach? :)
But, since mode check should not work on win32 platform (as far as i know), i've to return a fixed value.